wdl/tasks/Finalize.wdl

```diff
@@ -2,54 +2,10 @@ version 1.0
 
 import "Structs.wdl"
 
-task FinalizeToFile {
```

Comment:
Thought this can be removed since it's  not being used by any workflows. Also the functionality is embedded in the Array version